pykern/pkresource.py

```diff
@@ -8,34 +8,82 @@
 
 # Root module: Import only builtin packages so avoid dependency issues
 import errno
+import glob as builtin_glob
+import importlib
 import inspect
 import os.path
 import pkg_resources
 import re
 
+# TODO(e-carlin): discuss with rn if ok to import pkio
 from pykern import pkinspect
+from pykern import pkio
```

Comment:
@robnagler  is this ok?

Reply:
Yes, but add a comment about limiting imports in pkio.

Reply:
3260b72f49